packs/cloudflare/config.schema.yaml

```diff
@@ -0,0 +1,6 @@
+---
+api_host:
```

Comment:
This could actually just be a module level constant since this URL is unlikely to change any time soon, but anyway, I know you just updated the existing config :)